manifest_prod.yml

```diff
@@ -1,12 +1,16 @@
 ---
 memory: 512M
 stack: lucid64
+instances: 2
+env:
+  NEW_RELIC_CONFIG_FILE: newrelic.ini
+  NEW_RELIC_ENV: staging
 applications:
 - name: api
   path: .
   host: fec-prod-api
   env:
-    PRODUCTION: True
+    FEC_API_DEBUG: False
```

Comment:
Also, maybe it's worth thinking about a more generic approach to parsing environment variables. `configargparse` looks decent: https://pypi.python.org/pypi/ConfigArgParse